tests/visual/colorpicker.html

```diff
@@ -232,8 +232,73 @@
                 </div>
             </section>
 
-            <section></section>
-            <section></section>
+            <section>
+                <p>Angular colorpicker selected color tile</p>
+                <kendo-colorpicker class="k-widget k-colorpicker">
+                    <span class="k-picker-wrap k-state-default">
+                        <span class="k-selected-color" style="background-color: rgb(254, 65, 59);"></span>
+                    <span class="k-select">
+                        <span class="k-icon k-i-arrow-s"></span>
+                        </span>
+                    </span>
+                </kendo-colorpicker>
+            </section>
+
+            <section>
+                <p>Angular colorpicker with icon</p>
+                <kendo-colorpicker iconclass="k-icon k-i-edit-tools" class="k-widget k-colorpicker">
+                    <span class="k-picker-wrap k-state-default">
+                        <span class="k-tool-icon k-icon k-i-edit-tools">
+                            <span class="k-selected-color" style="background-color: rgb(254, 65, 59);"></span>
+                        </span>
+                        <span class="k-select">
+                            <span class="k-icon k-i-arrow-s"></span>
+                        </span>
+                    </span>
+                </kendo-colorpicker>
+            </section>
+
+            <section>
+                <p>Angular colorpicker disabled</p>
+                <kendo-colorpicker class="k-widget k-colorpicker">
+                    <span class="k-picker-wrap k-state-disabled">
+                        <span class="k-selected-color">
+                            <span class="k-icon k-i-line"></span>
+                        </span>
+                        <span class="k-select">
+                            <span class="k-icon k-i-arrow-s"></span>
+                        </span>
+                    </span>
+                </kendo-colorpicker>
+            </section>
+
+            <section>
+                <p>Angular colorpicker with opened colorpalette</p>
+                <kendo-colorpicker class="k-widget k-colorpicker">
+                    <span class="k-picker-wrap k-state-default">
+                        <span class="k-selected-color" style="background-color: rgb(78, 174, 197);"></span>
+                        <span class="k-select"><span class="k-icon k-i-arrow-s"></span></span>
+                    </span>
+                </kendo-colorpicker>
+                <kendo-popup class="k-animation-container k-animation-container-shown" style="left: 522px; top: 680px;">
```

Comment:
I'd suggest removing the `top` and `left` styles and use a static position for the popup. By doing this, the Bootstrap and Material tests would also look okay